package.json

```diff
@@ -47,30 +47,30 @@
     "@lezer/highlight": "^1.1.3",
     "@lit-labs/motion": "^1.0.2",
     "@lit-labs/virtualizer": "patch:@lit-labs/virtualizer@0.7.0-pre.2#./.yarn/patches/@lit-labs/virtualizer/event-target-shim.patch",
-    "@material/chips": "14.0.0-canary.261f2db59.0",
-    "@material/data-table": "14.0.0-canary.261f2db59.0",
-    "@material/mwc-button": "0.25.3",
-    "@material/mwc-checkbox": "0.25.3",
-    "@material/mwc-circular-progress": "0.25.3",
-    "@material/mwc-dialog": "0.25.3",
-    "@material/mwc-drawer": "^0.25.3",
-    "@material/mwc-fab": "0.25.3",
-    "@material/mwc-formfield": "0.25.3",
-    "@material/mwc-icon-button": "patch:@material/mwc-icon-button@0.25.3#./.yarn/patches/@material/mwc-icon-button/remove-icon.patch",
-    "@material/mwc-linear-progress": "0.25.3",
-    "@material/mwc-list": "^0.25.3",
-    "@material/mwc-menu": "0.25.3",
-    "@material/mwc-radio": "0.25.3",
-    "@material/mwc-ripple": "0.25.3",
-    "@material/mwc-select": "0.25.3",
-    "@material/mwc-slider": "0.25.3",
-    "@material/mwc-switch": "0.25.3",
-    "@material/mwc-tab": "0.25.3",
-    "@material/mwc-tab-bar": "0.25.3",
-    "@material/mwc-textarea": "^0.25.3",
-    "@material/mwc-textfield": "0.25.3",
-    "@material/mwc-top-app-bar-fixed": "^0.25.3",
-    "@material/top-app-bar": "14.0.0-canary.261f2db59.0",
+    "@material/chips": "^14.0.0",
+    "@material/data-table": "^14.0.0",
```

Comment:
these need to be pinned to the same version as used in `mwc`

Reply:
Fixed